### ISSUE TITLE
Quotation marks don't get stripped from strings

### ISSUE THREAD

**Issue by gamedevsam** [yhslai/haxetoml#5]
The beginning and end quotes should be stripped from strings.

**Comment by gamedevsam** [comment]
Here's a modified unescape function that takes care of this issue:

```
    function unescape(str : String) {
        var pos = 0;
        var buf = new haxe.Utf8();

        var len = Utf8.length(str);
        while(pos < len) {
            var c = Utf8.charCodeAt(str, pos);
            pos++;

            // strip first and last quotation marks
            if ((pos == 1 && c == "\"".code) ||
                (pos == len && c == "\"".code))
                continue;

            if(c == "\\".code) {
                c = Utf8.charCodeAt(str, pos);
                pos++;

                switch(c) {
                    case "r".code: buf.addChar("\r".code);
                    case "n".code: buf.addChar("\n".code);
                    case "t".code: buf.addChar("\t".code);
                    case "b".code: buf.addChar(8);
                    case "f".code: buf.addChar(12);
                    case "/".code, "\\".code, "\"".code: buf.addChar(c);
                    case "u".code:
                        var uc = Std.parseInt("0x" + Utf8.sub(str, pos, 4));
                        buf.addChar(uc);
                        pos += 4;
                    default:
                        throw("Invalid Escape");
                }
            } else {
                buf.addChar(c);
            }
        }

        return buf.toString();
    }
```

The important bit is this:

```
// strip first and last quotation marks
            if ((pos == 1 && c == "\"".code) ||
                (pos == len && c == "\"".code))
                continue;
```

**Comment by yhslai** [comment]
Fixed in f22eb85, thanks you!
